📖 book: extend next steps of quick start (#8252)
kubernetes-sigs/cluster-api: docs/book/src/user/quick-start.md

## Next steps

> Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.
> Deploying applications to your workload cluster - use the CNI deployment steps for pointers.<br>
> See the [clusterctl] documentation for more detail about clusterctl supported actions.

Reviewer: ```suggestion
- Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.
- Deploy applications to your workload cluster. Use the CNI deployment steps for pointers.<br>
- See the [clusterctl] documentation for more detail about clusterctl supported actions.
```

Add bullet points to make this a little bit clearer.

> 

Reviewer: ```suggestion
```

Remove line.

## Next steps

- Create a second workload cluster. Simply follow the steps outlined above, but remember to provide a different name for your second workload cluster.

> - Deploy applications to your workload cluster. Use the CNI deployment steps for pointers.<br>

Reviewer: ```suggestion
- Deploy applications to your workload cluster. Use the [CNI deployment steps](#deploy-a-cni-solution) for pointers.
```
